### ISSUE TITLE
Rendering bug with Dataview

### ISSUE THREAD

**Issue by taernsietr** [ckRobinson/multi-column-markdown#9]
Hey there! Just got the plugin and first of all thanks, I've been wanting something like this for a while :)

I'm trying to use the plugin in a note where I have two "blocks", one being a pure `HTML` table and the second a `dataview` table. So the setup looks like (changed ` for ' only here):

```
=== start-multi-column: <Main>  
'''column-settings  
number of columns: 2  
border: off
'''

<table>

=== end-column ===

'''dataview
TABLE
FROM "nanana"
'''

=== end-multi-column
```

The issue is, although the first table always renders just fine, the second one only seems to render if I make any change to the note (content, or editing column-settings). It will show the first time I open the note after making such a change, then stop appearing altogether if I open it a second time. Any further closing and reopening the note won't show the second table again unless I change something then reopen it, and the cycle then continues.

To clarify, the "column" does appear, but not any content from the `dataview` table (which, outside the column, always works just fine).

Edit: I've tried switching the order, `dataview` forst and `HTML` second, but the behavior continues.

**Comment by ckRobinson** [comment]
Hey thanks, I'm glad people have been finding it useful! 

As far as the bug goes: Whoops! Dataview was working in a past build so I must have accidentally missed testing it in one of the recent releases. This bug shares a root cause with the other open issue so I hope to get a fix out for both within the next day or so.

**Comment by ckRobinson** [comment]
This should be fixed as of the new 0.4.3 release. If you find any other bugs feel free to open another issue!

**Comment by taernsietr** [comment]
Just updated and verified that it seems to be fixed, although in my described case the dataview table still takes a few seconds to render. I won't open an issue or reopen this one, but if I do find something that seems a bit broken, I'll certainly report it :)

Thanks for the fix and for the work on the plugin!
